clang/lib/CodeGen/ItaniumCXXABI.cpp

```diff
@@ -4097,6 +4096,7 @@ llvm::Constant *ItaniumRTTIBuilder::BuildTypeInfo(
   // And the name.
   llvm::GlobalVariable *TypeName = GetAddrOfTypeName(Ty, Linkage);
   llvm::Constant *TypeNameField;
+  llvm::GlobalValue::UnnamedAddr Scope;
```

Comment:
`Scope` isn't a very clear name here. Maybe rename this to something like `UnnamedAddrScope`?